schema/apse/materials/materials_project/summary.json

```diff
@@ -0,0 +1,1265 @@
+{
```

Comment:
We could follow the convention for https://github.com/Exabyte-io/esse/tree/dev/schema/apse/db/nist_jarvis/2024.3.13 and use:

`apse/db/materials_project/2025.9.25/materials/summary.json`